charts/osdfir-infrastructure/README.md

```diff
@@ -0,0 +1,295 @@
+| `turbinia.gcp.gcpErrorReporting`                             | Enables GCP Cloud Error Reporting                                                                                                                                                                                    | `false`                                                                                      |
+
+### Turbinia Third Party
+
+
+### Redis Configuration
+
+| Name                                        | Description                                                                                  | Value            |
+| ------------------------------------------- | -------------------------------------------------------------------------------------------- | ---------------- |
+| `turbinia.redis.enabled`                    | Enables the Redis deployment                                                                 | `true`           |
+| `turbinia.redis.nameOverride`               | Overrides the Redis deployment name                                                          | `turbinia-redis` |
+| `turbinia.redis.master.count`               | Number of Redis master instances to deploy (experimental, requires additional configuration) | `1`              |
+| `turbinia.redis.master.persistence.size`    | Redis master Persistent Volume size                                                          | `8Gi`            |
+| `turbinia.redis.master.resources.limits`    | The resources limits for the Redis master containers                                         | `{}`             |
+| `turbinia.redis.master.resources.requests`  | The requested resources for the Redis master containers                                      | `{}`             |
+| `turbinia.redis.replica.replicaCount`       | Number of Redis replicas to deploy                                                           | `0`              |
+| `turbinia.redis.replica.persistence.size`   | Redis replica Persistent Volume size                                                         | `8Gi`            |
+| `turbinia.redis.replica.resources.limits`   | The resources limits for the Redis replica containers                                        | `{}`             |
+| `turbinia.redis.replica.resources.requests` | The requested resources for the Redis replica containers                                     | `{}`             |
+
+Specify each parameter using the --set key=value[,key=value] argument to helm install. For example,
+
+```console
+helm install my-release \
+    --set turbinia.enabled=false
+    oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/osdfir-infrastructure
+```
+
+The above command installs OSDFIR Infrastructure without Turbinia deployed.
+
+Alternatively, the `values.yaml` and `values-production.yaml` file can be 
```

Comment:
Should we disambiguate the different values.yaml (root dir vs. the ones in the turbinia/timesketch directories) and values-production.yaml files?

Reply:
I think for whatever the user pulls, they should see a single `values.yaml` and `values-production.yaml` per chart. So for example if they used the OSDFIR chart it'd have its own `values.yaml` and `values-production.yaml` (just added) and referencing the Turbinia/Timesketch equivalents wont work. The user also should be pulling from the OCI repo rather than this github repo so when they pull OSDFIR they'd get Turbinia/Timesketch tar'ed up and wouldn't directly see those files